deflect/qt/QmlStreamerImpl.cpp

```diff
@@ -293,6 +305,15 @@ bool QmlStreamer::Impl::_setupRootItem()
              [this]() { _updateSizes( QSize( _rootItem->width(),
                                              _rootItem->height( ))); } );
 
+    connect( _quickWindow, &QQuickWindow::minimumWidthChanged,
```

Comment:
Since we're moving to C++11, we could also go crazy and replace _sizeHintsChanged() by lambda functions here :-)